public/partials/report/modals/filter-prompt-modal.html

```diff
@@ -32,6 +32,35 @@ <h4 class="modal-title" translate>Runtime Filter properties</h4>
                 </div>
             </div>
         </div>
+
+        <div class="row" ng-if="selectedFilter.elementType !== 'date'">
+
+            <div class="col-md-8">
+                <label class="control-label" translate>Suggestions</label>
+                <div class="popover-primary ng-scope" popover-trigger="mouseenter" popover-placement="top" popover="{{ 'Select if you want suggest values on filter' | translate }}"></div>
```

Comment:
If it's not shown, I think it's better to remove the popover entirely.
And it's not correctly positioned anyway (it's on the div that contain 2 different inputs)

Reply:
popover removed :)